### DISCUSSION TITLE
¡Bienvenido al foro de homeassistant-edata!

### DISCUSSION THREAD

**Discussion by uvejota** [uvejota/homeassistant-edata#179] [Anuncios]
## 👋 ¡Bienvenido/a!
  He habilitado este foro para que podáis debatir acerca de lo que queráis. La idea es que podáis sugerir tarjetas, usos, configuraciones... o incluso compartir los problemas que tengáis con la API de Datadis, que como sabéis, escapan de nuestro control.

**Comment by powerlopez** [comment]
Primera sugerencia: Por favor habilita la posibilidad de tener una tarifa de 6 periodos, por ejemplo la 6.1TD. Para negocios y empresas nos vendría fenomenal.

Saludos y muy buen trabajo por todo, enhorabuena.

**Reply by uvejota** [reply]
Buenas, lo tengo en mente, pero primero voy a meterle mano al excedente, que me lo han pedido muchos usuarios domésticos.

**Comment by yagomayo** [comment]
Hola,
¿Podrías mejorar la documentación para introducir los parámetros de facturación? Estoy perdido con algunos campos.
En mi caso tengo endesa y busco los datos en la factura y algunos ni idea de qué son ni de donde sacarlos...

Gracias!
